frontend/server/src/Controllers/User.php

```diff
@@ -496,14 +493,15 @@ public static function apiMailingListBackfill(\OmegaUp\Request $r) {
         );
 
         foreach ($usersMissing as $user) {
-            $registered = self::registerToSendy($user);
+            $identity = \OmegaUp\DAO\Identities::findByUserId($user->user_id);
```

Comment:
```suggestion
            $identity = \OmegaUp\DAO\Identities::getByPK($user->main_identity_id);
```

igual en todos los demás lugares donde tienes el objeto `Users` disponible.

Reply:
Listo